index.js

```diff
@@ -63,8 +63,9 @@ function plugin(opts) {
 
         debug('adding %s to includes as `%s`', resolvedFilename, name);
 
-        file[name] = included[filename].contents;
-
+        file[name] = included[filename];
+        file[name].contents = included[filename].contents;
```

Comment:
That brings up a good point - does it make sense to clone the `included[filename]` object instead of just re-assigning it? I can foresee another plugin causing strange behavior they reference the same file and the included file isn't deleted.